app/questionnaire/placeholder_transforms.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from babel.numbers import format_currency
+from babel.dates import format_datetime
+
+
+class PlaceholderTransforms:
+    """
+    A class to group the transforms that can be used within placeholders
+    """
+    locale = 'en_GB'
```

Comment:
Shouldn't we using `flask_babel.get_locale()`?

Reply:
The intention here was that we don't have to hang everything from the application context that flask/flask_babel creates, where it's not really used for much. I've updated this to use default locale for now.